testing/fatutf8/fatutf8_main.c

```diff
@@ -166,5 +168,28 @@ int main(int argc, FAR char *argv[])
       exit(fd);
     }
 
+  printf("\n");
+
+  ret = remove(path);
+  if (ret == 0)
+    {
+      printf("removed %s\n", path);
+      ret = remove(dir_path);
```

Comment:
Maybe we can use next code instead of adding new variable to stack?
```
basepath = strrchr(path, '/');
if (basepath != NULL)
  {
    *basepath = '\0';
    ret = remove(path);
    if (ret == 0)
...
  }
```

Reply:
It is a good idea and has been revised